### PR TITLE
Drag/release frame text no longer collapses/expands (fixes #2554)

### LINKED ISSUE #2554: Selecting code in the detail view collapses box
Currently if one expands the code and then attempts to copy/paste code out of it, the box will collapse on releasing the mouse button.

### PR DESCRIPTION
**Pull request by benvinegar**
Not sure if `StrictClick` should have it's own `onClick` prop, or it should just override the top-most child's `onClick` prop. I chose the former ... but I'm okay with either.

/cc @getsentry/ui

### CODE REVIEW

File: src/sentry/static/sentry/app/components/strictClick.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    if (deltaX < StrictClick.MAX_DELTA_X && deltaY < StrictClick.MAX_DELTA_Y) {
+      this.props.onClick(evt);
+    }
+    this.setState({
+      startCoords: null
+    });
+  },
+
+  render() {
+    return React.cloneElement(this.props.children, {
```
Comment (mattrobenolt) [inline src/sentry/static/sentry/app/components/strictClick.jsx]: If this is cloning the children, does this end up with two layers in DOM? An `<ol>` inside an `<ol>`? Or am I just misunderstanding?

Comment (benvinegar) [inline src/sentry/static/sentry/app/components/strictClick.jsx]: You cannot replace props on an existing object. So this creates a clone (with new `onClick` and `onMouseMove` props) and renders that clone instead. It doesn't render two `<ol>` elements.

Comment (mattrobenolt) [inline src/sentry/static/sentry/app/components/strictClick.jsx]: Ah, clever.

**Comment by macqueen** [comment]
👍

**Pull request merged**